packages/compass-import-export/src/utils/get-shell-js.ts

```diff
@@ -30,21 +31,19 @@ export function newGetQueryAsShellJSString({
   query: ExportQuery;
 }) {
   let ret = `db.getCollection("${toNS(ns).collection}")`;
-  if (query.collation) {
-    ret += `.collate(\n  ${
-      stringify(query.collation as unknown as Record<string, unknown>) || ''
-    }\n)`;
-  }
   ret += `.find(\n`;
   ret += `  ${stringify(query.filter ? query.filter : {}) || ''}`;
   if (query.projection) {
     ret += `,\n  ${stringify(query.projection) || ''}`;
   }
   ret += '\n)';
+  if (query.collation) {
+    ret += `.collation(\n  ${stringify(query.collation as Document) || ''}\n)`;
```

Comment:
Updates how the collation looks:
| **before** | **after** |
| - | - |
| <img width="638" alt="Screenshot 2023-05-02 at 3 16 42 AM" src="https://user-images.githubusercontent.com/1791149/235603421-b0698391-3926-4fb0-92e8-7249540218d8.png"> | <img width="647" alt="Screenshot 2023-05-02 at 3 18 48 AM" src="https://user-images.githubusercontent.com/1791149/235603826-7c53ced2-83e8-44e7-9e4a-1f7908f3b8fc.png"> |

Reply:
How much work would it be go use consistent quotes? getCollection uses `"` and the rest uses `'`.

Reply:
Nice catch. Updated.